docs/controls/access_control/ac-02.md

```diff
@@ -0,0 +1,60 @@
+# Access Control
+## AC-02 - Account Management
+The organization:
+
+a. Identifies and selects the following types of information system accounts to support organizational missions/business functions: (individual, shared, group, system, guest/anonymous, emergency, developer/manufacturer/vendor, temporary, and service);
+
+b. Assigns account managers for information system accounts;
+
+c. Establishes conditions for group and role membership;
+
+d. Specifies authorized users of the information system, group and role membership, and access authorizations (i.e., privileges) and other attributes (as required) for each account;
+
+e. Requires approvals by (applicable information system managers) for requests to create information system accounts;
+
+f. Creates, enables, modifies, disables, and removes information system accounts in accordance with (the processes defined in the OS Standard Operating Procedures for Information Security: Access Control document (see Section AC-2));
+
+g. Monitors the use of information system accounts;
+
+h. Notifies account managers:
+
+   1. When accounts are no longer required;
+   2. When users are terminated or transferred; and
+   3. When individual information system usage or need-to-know changes;
+
+i. Authorizes access to the information system based on:
+
+   1. A valid access authorization;
+   2. Intended system usage; and
+   3. Other attributes as required by the organization or associated missions/business functions;
+
+j. Reviews accounts for compliance with account management requirements [at least every 180 days]; and
+
+k. Establishes a process for reissuing shared/group account credentials (if deployed) when individuals are removed from the group.
+
+For CSP Only: AC-2 (j) [at least annually]
+
+## TDP Implementation
+a. The TDP system accounts are OFA Admin and Data Prepper.  
+
+b. The OFA Admin is responsible for managing the system accounts.  The OFA Admin approves new users, updates profile information, deactivates, and reactivates users.
```

Comment:
Based on the conversation [here](https://github.com/HHS/TANF-app/pull/174#discussion_r573264892) this doesn't seem to be accurate - it is the `System Owner` that would manage accounts, not `OFA Admins` - is that correct?

Reply:
If developers are included in part `b.`, we should also clarify who provides oversight/access for dev accounts.

Reply:
This will be true down the line, but for OFA MVP only the System Owner will do these things

Reply:
Updated per our discussion on Friday to change this to System Admin.